pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -81,6 +85,46 @@ func (c *defaultPVCHandler) CreatePVCsForWorkspaces(ctx context.Context, wb []v1
 	return errorutils.NewAggregate(errs)
 }
 
+func (c *defaultPVCHandler) PurgeFinalizerAndDeletePVCForWorkspace(ctx context.Context, pvcName, namespace string) error {
+	p, err := c.clientset.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvcName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get the PVC %s: %w", pvcName, err)
+	}
+
+	// get the list of existing finalizers and drop `pvc-protection` if exists
+	var finalizers []string
+	for _, f := range p.ObjectMeta.Finalizers {
+		if f == "kubernetes.io/pvc-protection" {
```

Comment:
nit: Is there a constant defined for this value in k8s libs anywhere?

Reply:
The constant is defined in https://github.com/kubernetes/kubernetes/blob/dde22b3a5ede36bfaec97827ade95df69467959d/pkg/volume/util/finalizer.go#L21, which is `k8s.io/kubernetes` pkg. It seems this pkg is not recommend to be imported directly